test/AffInterface.purs

```diff
@@ -94,207 +81,65 @@ suite = do
     test "Can process GetDatumsByHashes" do
       testOgmiosDatumCacheGetDatumsByHashes
 
-testOgmiosDatumCacheGetDatumByHash :: Aff Unit
-testOgmiosDatumCacheGetDatumByHash = runQueryM testnetTraceQueryConfig do
+testOgmiosDatumCacheGetDatumByHash :: QueryM Unit
+testOgmiosDatumCacheGetDatumByHash = do
   -- Use this to trigger block fetching in order to actually get the datum:
   -- ```
   -- curl localhost:9999/control/fetch_blocks -X POST -d '{"slot": 54066900, "id": "6eb2542a85f375d5fd6cbc1c768707b0e9fe8be85b7b1dd42a85017a70d2623d", "datumFilter": {"address": "addr_xyz"}}' -H 'Content-Type: application/json'
   -- ```
   void $ getDatumByHash $ DataHash $ hexToByteArrayUnsafe
     "f7c47c65216f7057569111d962a74de807de57e79f7efa86b4e454d42c875e4e"
 
-testOgmiosDatumCacheGetDatumsByHashes :: Aff Unit
-testOgmiosDatumCacheGetDatumsByHashes =
-  runQueryM testnetTraceQueryConfig do
-    -- Use this to trigger block fetching in order to actually get the datum:
-    -- ```
-    -- curl localhost:9999/control/fetch_blocks -X POST -d '{"slot": 54066900, "id": "6eb2542a85f375d5fd6cbc1c768707b0e9fe8be85b7b1dd42a85017a70d2623d", "datumFilter": {"address": "addr_xyz"}}' -H 'Content-Type: application/json'
-    -- ```
-    _datums <- getDatumsByHashes $ pure $ DataHash $ hexToByteArrayUnsafe
-      "f7c47c65216f7057569111d962a74de807de57e79f7efa86b4e454d42c875e4e"
-    pure unit
+testOgmiosDatumCacheGetDatumsByHashes :: QueryM Unit
+testOgmiosDatumCacheGetDatumsByHashes = do
+  -- Use this to trigger block fetching in order to actually get the datum:
+  -- ```
+  -- curl localhost:9999/control/fetch_blocks -X POST -d '{"slot": 54066900, "id": "6eb2542a85f375d5fd6cbc1c768707b0e9fe8be85b7b1dd42a85017a70d2623d", "datumFilter": {"address": "addr_xyz"}}' -H 'Content-Type: application/json'
+  -- ```
+  _datums <- getDatumsByHashes $ pure $ DataHash $ hexToByteArrayUnsafe
```

Comment:
Again, I know this wasn't introduced in your commits but stylistically I'd prefer to use `void` here, then we don't need the `pure unit` either